### ISSUE TITLE
Allow building with Python 3

### ISSUE THREAD

**Issue by AtnNn** [rethinkdb/rethinkdb#3731]
The configure script complains:

```
* Error: python 3.2.3 is too recent. At most python 2.9.99 is required.
```

However, if that restriction is removed, RethinkDB seems to build fine with a few minor tweaks to the python scripts.

**Comment by AtnNn** [comment]
@deontologician has fixed a few python 3 incompatibilities in our scripts.

However, building v8 depends on python 2. Upgrading to a newer v8 (#3423) might help.

**Comment by AtnNn** [comment]
Closed in 7565b5b.

@anatol This will be in RethinkDB 2.0
